Samples/PowerShell/ShardManagement/CreatePerformanceCounterCategory.ps1

```diff
@@ -0,0 +1,43 @@
+﻿<#
+/********************************************************
+*                                                        *
+*   © Microsoft. All rights reserved.                    *
+*                                                        *
+*********************************************************/
+
+.SYNOPSIS
+    This script creates performance counter catagory and adds
+    specified user to 'Performance Monitor Users' local group.
+
+.NOTES
+    Author: Microsoft SQL Elastic Scale team
+    Last Updated: 5/6/2016
+
+.EXAMPLES
+    .\CreatePerformanceCounterCategory.ps1 `
+        -UserName 'domain\username'
+#>
+
+param(
+	[Parameter(Mandatory = $true)]
+	[string]
+	$UserName=""
+)
+$scriptPath = split-path -parent $MyInvocation.MyCommand.Path
+$dllNameWithPath = $scriptPath + "\Microsoft.Azure.SqlDatabase.ElasticScale.Client.dll";
```

Comment:
The import will fail if the dll is missing and the user refuses to download it, so once you add that line you don't need the 'if' check below and you can also remove the 'else' block